var/spack/repos/builtin/packages/mpifileutils/package.py

```diff
@@ -3,10 +3,8 @@
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
 
-from spack import *
```

Comment:
You probably don't want to delete this line

Reply:
I actually did. That line has been unnecessary for a while, since #8090. But @tgamblin worked out the fix so it would be backwards compatible so we wouldn't have to update all the packages.